maggy/core/rpc.py

```diff
@@ -1,7 +1,6 @@
-
 import threading
 import struct
-import pickle
+import cloudpickle
```

Comment:
Use PySpark's cloudpickle, to avoid having to add cloudpickle additionally to the conda base environment.
```suggestion
from pyspark import cloudpickle
```